src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int NUMBER_OF_FIGURES_TYPES = 5;
+    static final int MAX_VALUE = 10;
+
+    public Figure getRandomFigure() {
+        Random figureNumber = new Random();
+        Random figurePart = new Random();
```

Comment:
```suggestion
        Random random = new Random();
```
you can use the same random for different purposes - let's create only one variable